webapp/src/components/musicEditor/playback.ts

```diff
@@ -193,4 +193,42 @@ export function addPlaybackStopListener(listener: () => void) {
 
 export function removePlaybackStopListener(listener: () => void) {
     playbackStopListeners = playbackStopListeners.filter(l => listener !== l);
-}
\ No newline at end of file
+}
+
+const workerJS = `
+/**
+ * Adpated from https://github.com/cwilso/metronome/
```

Comment:
License request done?

Reply:
I didn't actually use any of their code, more just the idea. I can do the license request anyway to be safe.